package.json

```diff
@@ -20,12 +20,16 @@
     "prepublish": "grunt test"
   },
   "devDependencies": {
+    "async": "^0.9.0",
```

Comment:
These are all required by freedom-for-chrome's integration task (https://github.com/freedomjs/freedom-for-chrome/blob/master/tasks/integration.js)